pkg/scheduler/plugins/predicates/helper_test.go

```diff
@@ -17,19 +17,19 @@ limitations under the License.
 package predicates
 
 import (
-	"testing"
-
-	"github.com/stretchr/testify/assert"
 	utilfeature "k8s.io/apiserver/pkg/util/feature"
 	featuregatetesting "k8s.io/component-base/featuregate/testing"
 	"k8s.io/kubernetes/pkg/features"
+	"testing"
```

Comment:
Need to adjust the order of packages for the test file as well

Reply:
done.